package.json

```diff
@@ -5,11 +5,12 @@
   "description": "Red Badger's website",
   "main": "build/server.js",
   "scripts": {
+    "clean": "rimraf build",
     "test:server": "BABEL_DISABLE_CACHE=1 BABEL_ENV=test mocha --compilers js:babel-register ./src/**/*.spec.js",
     "test:client": "karma start config/karma.config.js",
     "test:e2e": "wdio config/wdio.config.js",
     "start": "node build/server.js",
-    "build": "webpack --config config/webpack.config.js",
+    "build": "npm run clean && webpack --config config/webpack.config.js",
```

Comment:
It will run everywhere. It's just that webpack won't remove old files from the build folder so old ones will just be hanging around. Given that we're using `require` to bring in CSS, images, etc. I just thought we might get weird results at runtime or even in tests.

I can just make it `rm -rf build` I suppose. I'm just used to having to consider the fact that sometimes you'll need to build on Windows :)